diff --git a/fiduswriter/base/default_settings.py b/fiduswriter/base/default_settings.py
index c22abf9c..5c6878b8 100755
--- a/fiduswriter/base/default_settings.py
+++ b/fiduswriter/base/default_settings.py
@@ -1,13 +1,13 @@
 import os

-PORT = 8000
+PORT = 4386

 # If you want to show debug messages, set DEBUG to True.
-DEBUG = True
+DEBUG = False

 # This determines whether the server is used for testing and will let the
 # users know upon signup know that their documents may disappear.
-TEST_SERVER = True
+TEST_SERVER = False
 # This is the contact email that will be shown in various places all over
 # the site.
 CONTACT_EMAIL = 'mail@email.com'
@@ -27,14 +27,25 @@ PROJECT_PATH = os.environ.get('PROJECT_PATH')
 # SRC_PATH is the root path of the FW sources.
 SRC_PATH = os.environ.get('SRC_PATH')

+SNAP_DATA = os.environ.get('SNAP_DATA')
+DATABASE_PASSWORD = ''
+PASSWORD_PATH = '{}/mysql/fiduswriter_password'.format(SNAP_DATA)
+if os.path.isfile(PASSWORD_PATH):
+    with open(PASSWORD_PATH, 'r') as f:
+        DATABASE_PASSWORD = f.read().strip()
+
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(PROJECT_PATH, 'fiduswriter.sql'),
-        'CONN_MAX_AGE': 15
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'fiduswriter',
+        'USER': 'fiduswriter',
+        'PASSWORD': DATABASE_PASSWORD,
+        'HOST': '/tmp/sockets/mysql.sock'
     }
 }

+AUTO_SETUP = True
+
 # Will let any user not delete more than 5000 bibliography entries
 # simultaneously
 DATA_UPLOAD_MAX_NUMBER_FIELDS = 5000
@@ -101,7 +112,7 @@ MEDIA_URL = '/media/'
 # Don't put anything in this directory yourself; store your static files
 # in apps' "static/" subdirectories and in STATICFILES_DIRS.
 # Example: "/home/media/media.lawrence.com/static/"
-STATIC_ROOT = '/tmp/django-static/'
+STATIC_ROOT = os.path.join(PROJECT_PATH, 'static-collected/')
 # URL prefix for static files.
 # Example: "http://media.lawrence.com/static/"
 STATIC_URL = '/static/'
